jitsu/helm/jitsu/templates/deployment.yaml

```diff
@@ -57,6 +57,11 @@ spec:
             periodSeconds: 5
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+        - name: dind-daemon
```

Comment:
I think to run docker in docker on a cloud k8s deployment, you need more since the host docker socket isn't exposed.   I've done this before and think you need something like:

```
 securityContext:
    privileged: true
  env:
  - name: DOCKER_TLS_CERTDIR
    value: /certs
  volumeMounts:
  - name: share
    mountPath: /tmp/shared
  - name: docker-storage
    mountPath: /var/lib/docker
  - name: docker-certs
    mountPath: /certs
volumes:
- name: share
  emptyDir: {}
- name: docker-storage
  emptyDir: {}
- name: docker-certs
  emptyDir: {}
```

And then the other container will need to set a DOCKER_HOST env var to tcp://localhost:2375

Reply:
we'd probably want this to be optional too, since some people won't want the security issues with privileged containers/dind generally

Reply:
@michaeljguarino Thanks for providing the configuration part.
Some parts of the configuration were updated.
Testing on AWS showed that it is working.

Reply:
the certs dir didn't need to be mounted?

Reply:
It is an interesting question. I tested only our purposes.
For our purposes, I did not find issues with certificates without that directory.
But maybe there is some behaviour that is out of our scope.

Reply:
New options for the new capability were added.